doc/Command-Reference.md

```diff
@@ -1730,6 +1734,76 @@ This command is used to remove particular IPv4 or IPv6 BGP neighbor configuratio
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#bgp)
 
+## Container Auto-restart
+SONiC introduced a feature about auto-restarting docker containers
+if one of critical processes accidently crashed or exited. The motivation behind
+this feature is that if one critical process in a docker container crashed, then
+the whole container will not work correctly and we can try to restart to make 
+it enter into the healthy state.
+
+### Container Auto-restart show commands
+
+**show container feature autorestart**
+
+This command will display the status of auto-restart feature for all containers.
+
+- Usage:
+  ```
+  show container feature autorestart
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ show container feature autorestart
+  Container Name    Status
+  --------------    --------
+  database          enabled
+  syncd             enabled
+  teamd             disabled
+  dhcp_relay        enabled
+  lldp              enabled
+  pmon              enabled
+  bgp               enabled
+  swss              disabled
+  telemetry         enabled
+  sflow             enabled
+  snmp              enabled
+  radv              disabled
+  ```
+
+**show container feature autorestart <container_name>**
+
+This command displays the status of auto-restart feature for a specific container.
+
+- Usage:
+  ```
+  show container feature autorestart pmon
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ show container feature autorestart database
+  Container Name    Status
+  --------------    --------
+  database          enabled
+  ```
```

Comment:
No need for this section, as it's not a different command, it's just a variation on the command above. As with other commands in this file, you can simply replace this section with the following:


Optionally, you can specify a container name in order to display the auto-restart feature status for that container only

- Example:
  ```
  admin@sonic:~$ show container feature autorestart database
  ```

Reply:
Good suggestions! Please review it again.